feat: make action more flexible and fix an edge case (#2)
fmaule/generate-build-manifest: README.md

  action-info: false # disable writing action information to manifest (just an example)
```

Provide a custom path for the Dockerfile

```yaml
uses: fmaule/generate-build-manifest@v2
with:

>   # imagining you have a libs folder with the service inside of it 

Reviewer: ```suggestion
  # assuming you have a 'libs' folder that includes the service
```
